build.gradle

```diff
@@ -91,5 +91,5 @@ subprojects {
 }
 
 ext {
-    fluxCVersion = 'a3864fc925e563e74028510b492be8dee80e45f9'
+    fluxCVersion = '9ecf17d9361959c540511a7da2ed0163c2476d74'
```

Comment:
just passing by - should this be updated to latest commit in https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/1059/commits/c560d7e602271ac85d7b3d5253fe3b8614a8ca7d ?

Reply:
I don't think so but I've merged the last develop in FluxC to my branch and I'm updating it here too so the logic should be there. Once we merge the FluxC branch, we can update this hash code.